include/osw_hal.h

```diff
@@ -66,6 +66,7 @@ class OswHal {
 
     // Buttons (Engine-Style)
     bool btnHasGoneDown(Button btn);
+    bool btnDoublePress(Button btn);
```

Comment:
maybe better: btnIsDoubleClick ?

the PR talks about double-click, the code about press.